### DISCUSSION TITLE
Receber pagamento na zona do EURO via SEPA e não por SWIFT

### DISCUSSION THREAD

**Discussion by DiegoMoussallem** [DyegoCosta/trabalhando-remoto#166] [General]
Olá, 

estou enfrentando um problema que meu pagador não quer fazer transferências via SWIFT e está me pedindo uma conta SEPA, ou seja, um IBAN dentro da Zona do Euro. O Wise possui e outras plataformas também, mas somente para PF. Alguém conseguiria me ajudar e saberia dizer se existe alguma empresa como a Husky, Remessa Online que tenha um IBAN na zona do Euro (SEPA)?

**Comment by hamagust** [comment]
Ola. Uso conta payoneer em EURO. Se alguém tiver uma alternativa, estou
interessada também...

Em ter, 24 de mai de 2022 13:14, Diego Moussallem ***@***.***>
escreveu:

**Comment by DiegoMoussallem** [comment]
@hamagust mas a payoneer é para PF tb certo?
Igual a WISE.

**Comment by hamagust** [comment]
Para ambos, PF e PJ...

Em ter., 24 de mai. de 2022 às 13:54, Diego Moussallem <
***@***.***> escreveu:
